debian/copyright

```diff
@@ -0,0 +1,33 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: sanoid
+Source: <https://github.com/jimsalterjrs/sanoid>
+
+Files: *
+Copyright: 2017 Jim Salter <github@jrs-s.net>
+License: GPL-3.0+
+
+Files: debian/*
+Copyright: 2017 Jim Salter <github@jrs-s.net>
```

Comment:
(Since the changes in the debian/ directory are small, I'm comfortable assigning copyright to you)

Reply:
Thanks for your work on this!